src/genomedata_load_data.c

```diff
@@ -117,6 +117,42 @@ typedef struct {
 #endif /* GCC_VERSION >= 20500*/
 #endif /* __GNUC__ */
 
+void load_data(char* gdfilename, char* trackname, bool verbose);
+
+static PyObject* load_data_from_stdin(PyObject* self, PyObject* args) {
+  const char* gdfilename;
+  const char* trackname = "dummy";
```

Comment:
Why `"dummy"`?

Reply:
I think I accidentally reverted this line before the commit. Practically it makes no difference since it gets ignored by ParseTuple but I will remove it.